Account/SyncAuthState.swift

```diff
@@ -53,6 +54,9 @@ extension SyncAuthStateCache: JSONLiteralConvertible {
 open class FirefoxAccountSyncAuthState: SyncAuthState {
     fileprivate let account: FirefoxAccount
     fileprivate let cache: KeychainCache<SyncAuthStateCache>
+    public var deviceRegistration: FxADeviceRegistration? {
+        return account.deviceRegistration
```

Comment:
… and then here, `account.deviceRegistration?.id`

Reply:
👍